libraries/AP_IQ_Motor/AP_IQ_Motor.cpp

```diff
@@ -25,13 +31,6 @@ const AP_Param::GroupInfo AP_IQ_Motor::var_info[] = {
     // @User: Advanced
     AP_GROUPINFO("TELEM", 2, AP_IQ_Motor, _telemetry_bitmask, 0),
 
-    // @Param: MOT_DIR
-    // @DisplayName: Motor Direction Bitmask
-    // @Description: This sets the direction of each motor. 0 is CCW, 1 is CW.
-    // @Range: 0 65535
-    // @User: Advanced
-    AP_GROUPINFO("DIR", 3, AP_IQ_Motor, _motor_dir_bitmask, 0),
```

Comment:
I'm a bit out of the loop, so I may have some obvious questions. Is this just a vestigial thing that has no purpose anymore?

Reply:
Yep, the motor direction bitmask doesn't actually get used anywhere in our control through ardupilot